src/main/java/nl/b3p/tailormap/api/persistence/helper/ApplicationHelper.java

```diff
@@ -252,6 +252,9 @@ private boolean addAppLayerItem(AppTreeLayerNode layerRef) {
         attribution = service.getSettings().getDefaultLayerSettings().getAttribution();
       }
 
+      String description = appLayerSettings.getDescription();
+      layerRef.setDescription(description);
```

Comment:
This works differently from other settings and introduces a side effect to this method by changing the `AppTreeLayerNode` property of the json property of the entity, which imo should be read-only. In the `addAppTreeNodeItem()` method the description of the frontend-model `LayerTreeNode` is set from the `AppTreeLayerNode` description so I understand this works, but I feel there is a mismatch with where the frontend expects the description and where it is convenient in the admin to save it as a setting.

Reply:
Agree that it is a side-effect. 
Alternatively it could be stored/submitted/communicated on a `layerNode` from the admin instead of   a `layerSetting` as it is now

Reply:
Made description an app layer, layer and default layer setting.